mentat/session_input.py

```diff
@@ -57,9 +57,10 @@ async def collect_input_with_commands() -> StreamMessage:
     response = await collect_user_input()
     while isinstance(response.data, str) and response.data.startswith("/"):
         try:
-            arguments = shlex.split(response.data[1:])
-            command = Command.create_command(arguments[0])
-            await command.apply(*arguments[1:])
+            # We only use shlex to split the arguments, not the command itself
+            arguments = shlex.split(" ".join(response.data.split(" ")[1:]))
```

Comment:
`split` has a maximum number arg which you can use instead of splitting and rejoining. e.g. `response.data.split(" ", 1)[1]` is equivalent to this argument.

Reply:
I was using that but then I have to check if the list is empty (since [1] returns index out of range error) and I eventually decided that this was cleaner